### ISSUE TITLE
Progress bars are missing ARIA tags and screen reader tags

### ISSUE THREAD

**Issue by Nicd** [Nicd/code-stats#9]
Progress bars should have tags for screen readers (`<span class="sr-only">`) and ARIA tags to be accessible.
